### PR TITLE
Comment with preview

### PR DESCRIPTION
**Pull request by annatraussnig**
Screenshot 2020-11-30 at 17 46 52

### Also in this PR

- dark mode for active debates teaser

TODO: 

- [x] fix broken links
- [x] tweak workflow to feature comments in FE/BE

### CODE REVIEW

File: src/components/CommentTeaser/CommentTeaser.js [hunk 1]
```diff
@@ -182,16 +195,49 @@ export const CommentTeaser = ({
             </Link>
           </CommentBodyParagraph>
         </div>
+
+        {documentPreview && (
```
Comment (ovbm) [inline src/components/CommentTeaser/CommentTeaser.js:199]: Question: This changes all comment teasers to add an image if they have a twitter image. But in an element like the discussion section on the main feed, we wouldn't want images there. Should there be a prop "withPreview"? Or does the query there not ask for "document meta twitterImage" and thus it's ok to not use a prop?
Screenshot 2020-12-04 at 11 08 14

File: src/components/Discussion/Internal/Comment/IconLink.js [hunk 2]
```diff
@@ -62,18 +56,34 @@ export const IconLink = ({
...
-    <a href={href} onClick={onClick} {...styles.link} style={patchedStyle}>
+    <a
+      href={href}
+      onClick={onClick}
+      {...styles.link}
+      {...linkStyleRule}
+      style={patchedStyle}
+    >
       <span {...styles.icon}>
-        <Icon size={dimension} {...colorScheme.set('fill', 'primary')} />
```
Comment (ovbm) [inline src/components/Discussion/Internal/Comment/IconLink.js:69]: why remove fill here?

Comment (annatraussnig) [reply src/components/Discussion/Internal/Comment/IconLink.js:69]: The color is now set through the `linkStyleRule`. It includes a hover state and fixes the hover color, which was different from the other links.

File: src/components/CommentTeaser/CommentTeaser.js (hunk 1 above)
Comment (annatraussnig) [reply src/components/CommentTeaser/CommentTeaser.js:199]: You make a good point. Using the twitter image implicitly is weird.

File: src/components/Discussion/Internal/Comment/IconLink.js (hunk 2 above)
Comment (ovbm) [reply src/components/Discussion/Internal/Comment/IconLink.js:69]: 👍🏼

**Review by ovbm — APPROVED** [review]

**Pull request merged**